.github/workflows/analyze-dependents.yml

```diff
@@ -280,9 +280,9 @@ jobs:
       uses: hmarr/auto-approve-action@v2
       with:
         pull-request-number: ${{ steps.cpr.outputs.pull-request-number }}
-        github-token: ${{ secrets.requirements_bot_github_token }}
+        github-token: ${{ secrets.EDX_NETLIFY_PAT }}
```

Comment:
[curious] Are these changes remnants of dealing with issues in the Git history? Let's keep this using `secrets.requirements_bot_github_token` moving forward as that's what's on `master`. The EDX_NETLIFY_PAT secret is my hack before I knew `requirements_bot_github_token` existed :)

Reply:
Roll-backed this commit in this pull request